CodeReview/src/commands/CCodeReviewComment.cpp

```diff
@@ -49,12 +54,24 @@ bool CCodeReviewComment::canInterpret(Visualization::Item*, Visualization::Item*
 	return false;
 }
 
-Interaction::CommandResult* CCodeReviewComment::execute(Visualization::Item*, Visualization::Item*,
-				const QStringList&, const std::unique_ptr<Visualization::Cursor>& cursor)
+Interaction::CommandResult* CCodeReviewComment::execute(Visualization::Item* source, Visualization::Item*,
+				const QStringList&, const std::unique_ptr<Visualization::Cursor>&)
 {
-	auto cursorOwner = cursor->owner();
-	auto overlay = new CodeReviewCommentOverlay{cursorOwner};
-	cursorOwner->addOverlay(overlay, "CodeReviewComment");
+	for (auto manager : Model::AllTreeManagers::instance().loadedManagers())
+		if (manager->nodeIdMap().contains(source->node()))
```

Comment:
Like above, do not use contains since then you'll do the same work twice in `id`. Implement a different method that only returns the node if it doesn't exist and returns `nullptr` otherwise (unlike the current `id`) make sure the current `id` uses that other method.